package.json

```diff
@@ -84,5 +84,6 @@
       "prettier --write",
       "eslint --fix"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
😨 let's just remove this one. everything else is perfect now

Reply:
Damn, didn't realize it came back.
I removed it now.